client/js/location.js

```diff
@@ -0,0 +1,9 @@
+"use strict";
+
+// This is a thin wrapper around `window.location`, in order to contain the
+// side-effects. Do not add logic to it as it cannot be tested, only mocked.
+module.exports = {
+	reload() {
+		window.location.reload();
```

Comment:
Do you want to update all `location.reload` references in Lounge to use this wrapper?

I noticed some of them also use `reload(true)` to force a reload.

Reply:
Yeah will do, but it turns out there is no other instance that can use this one. I'll add `forceReload` (`reload(true)`) in another PR because I am _trying_ to add some test around it, which is times harder that I expected...